.github/workflows/ci.yaml

```diff
@@ -4,11 +4,11 @@ on:
     types: [opened, synchronize, reopened]
   push:
     branches:
-    - master
+    - '*'
+    - '!dependabot/**'
 
 env:
-  IMAGE_NAME: "equinix/cloud-provider-equinix-metal"
-
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
This allows for forks to also be able to build images and use the workflows.

Reply:
Nice. When it matches, this is much easier.